pkg/arvo/app/chat-view.hoon

```diff
@@ -156,25 +156,40 @@
     =/  group-read=path  [%chat (weld pax /read)]
     =/  group-write=path  [%chat (weld pax /write)]
     :_  this
-    %+  weld
-      :~  (chat-poke [%create our.bol path.act])
-          (group-poke [%bundle group-read])
-          (group-poke [%bundle group-write])
-          (group-poke [%add read.act group-read])
-          (group-poke [%add write.act group-write])
-          (chat-hook-poke [%add-owned pax security.act])
-      ==
-    (create-security [%chat pax] security.act)
+    %-  zing
+    :~  :~  (group-poke [%bundle group-read])
+            (group-poke [%bundle group-write])
+            (group-poke [%add read.act group-read])
+            (group-poke [%add write.act group-write])
+            (chat-poke [%create our.bol path.act])
+            (chat-hook-poke [%add-owned pax security.act])
+        ==
+        (create-security [%chat pax] security.act)
+        :~  (permission-hook-poke [%add-owned group-read group-read])
+            (permission-hook-poke [%add-owned group-write group-read])
+        ==
+    ==
```

Comment:
Can't you make these a flat list in the code, instead of `zing`ing?

Reply:
No, because (create-security) returns a list.

Reply:
Oops, of course. If order around that doesn't matter too much you could do a `:*` with `create-security` at the end. If it does, then out of luck for this I guess.

Reply:
Order does matter, unfortunately.